### PR TITLE
Removed an empty line

### PR DESCRIPTION
**Pull request by vijayVijeyender**

### CODE REVIEW

File: src/main/java/com/employeeManagement/employee/Employee.java [hunk 1]
```diff
@@ -10,7 +10,6 @@
 
 @Entity
 public class Employee {
-    
     public Employee() {
```
Comment (vijayVijeyender) [inline src/main/java/com/employeeManagement/employee/Employee.java:13]: hello

File: src/main/java/com/employeeManagement/employee/Employee.java [hunk 2]
```diff
@@ -10,7 +10,6 @@
 
 @Entity
 public class Employee {
-
```
Comment (vijayVijeyender) [inline src/main/java/com/employeeManagement/employee/Employee.java:13]: i need this empty line please don't delete it

Comment (Kavimaha123) [reply src/main/java/com/employeeManagement/employee/Employee.java:13]: ok

**Comment by vijayVijeyender** [comment]
By mistake

**Pull request closed without merging**